config/global-setup.js

```diff
@@ -0,0 +1,19 @@
+const http = require("http")
+const fs = require("fs")
+const path = require("path")
+const setup = require("jest-playwright-preset/setup")
+
+const requestListener = (html) => (_, res) => {
+  res.writeHead(200)
+  res.end(html)
+}
+
+module.exports = async (config) => {
```

Comment:
(I think if you want, you can even write the setup and teardown in TypeScript)

Reply:
Ooh, nice!

Reply:
Updated!